<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Libkface GUI Test Program</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Sensitivity:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="3">
       <widget class="QSlider" name="sensitivitySlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>accurate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Accuracy:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>more false positives</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>more missed faces</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QSlider" name="accuracySlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>fast</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="7">
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Detection</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="openImageBtn">
        <property name="text">
         <string>Open Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="detectFacesBtn">
        <property name="text">
         <string>Detect Faces</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recogniseBtn">
        <property name="text">
         <string>Recognize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateDatabaseBtn">
        <property name="text">
         <string>Update Database</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="configLocation">
      <property name="text">
       <string>Default database file open</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
